docs/integration/rabbitmq-logs.md

```diff
@@ -0,0 +1,72 @@
+title: RabbitMQ Logs Integration
+description: Sematext RabbitMQ Logs integration allows you to troubleshoot connection, access, and other RabbitMQ issues.
+
+To make use of the Sematext RabbitMQ Logs integration, you'll need to send parsed RabbitMQ logs to your Sematext Logs App. The easiest way is via [Logagent](../logagent/index.md), which can parse RabbitMQ logs out of the box.
+
+Once data is in, you can explore it via the built-in reports: 
+
+<img
+  class="content-modal-image"
+  alt="RabbitMQ Logs Overview"
+  src="../../images/agents/rabbitmq_overview.png"
+  title="RabbitMQ Logs Overview"
+/>
+
+## Setting up Logagent
+
+With [Node.js installed](https://nodejs.org/en/download/package-manager/), you'd first need to install Logagent:
+```bash
+sudo npm i -g @sematext/logagent
+```
+
+Then, write a config file that tails your RabbitMQ logs and sends them to your RabbitMQ Logs App. Parsing happens out of the box, since we match the `rabbit` source:
```

Comment:
Is it "rabbit" or "rabbitmq"?

Reply:
it is `rabbit` because logs can look like `rabbit@ubuntu2004.log`